BuildingBlocks/src/BuildingBlocks.Infrastructure/EventBus/RabbitMQ/EventBusRabbitMQ.cs

```diff
@@ -169,7 +169,7 @@ private IModel CreateConsumerChannel()
             }
             catch (Exception ex)
             {
-                channel.BasicReject(eventArgs.DeliveryTag, true);
+                channel.BasicReject(eventArgs.DeliveryTag, false);
```

Comment:
is this intended?

Reply:
I hate boolean params :(

Reply:
Yes. See PR description.

Reply:
> I hate boolean params :(

Could have used a named parameter. But it's too late now. Don't want to wait for another round of pipelines. ;)